test/index.js

```diff
@@ -94,6 +94,28 @@ describe('KeyringController', function () {
     });
   });
 
+  describe('persistAllKeyrings', function () {
+    it('should persist keyrings in unsupportedKeyrings array', async function () {
+      const unsupportedKeyring = 'DUMMY_KEYRING';
+      keyringController.unsupportedKeyrings = [unsupportedKeyring];
+      await keyringController.persistAllKeyrings();
```

Comment:
Nit: typically for tests, we use newlines to separate the different phases of the test: arrange, act, and assert. Arrange is the setup, act is where we do the thing that is under test, and assert is where we check that it works.

Adding newlines to separate these sections would make this easier to read:

```suggestion

      await keyringController.persistAllKeyrings();
  
```

Likewise for the other two tests